src/bench/coin_selection.cpp

```diff
@@ -36,12 +36,7 @@ static void CoinSelection(benchmark::State& state)
     std::vector<COutput> vCoins;
     LOCK(wallet.cs_wallet);
 
-    while (state.KeepRunning()) {
-        // Empty wallet.
-        for (COutput output : vCoins)
-            delete output.tx;
-        vCoins.clear();
-
+    do {
```

Comment:
I'm not sure I understand the rationale for going to `while(...)` to `do {...} while(...)` as used in all the other benchmarks. Say this function is called with `state.KeepRunning()` already zero, does it still need to do one iteration?

Reply:
Right, just moving the cleanup below is enough right?